front/src/components/RegistrationUser/RegistrationUser.component.html

```diff
@@ -0,0 +1,73 @@
+  <div>   
+    <el-button type="primary" @click="VisibleRegistrationForm(true)">Зарегистрироваться</el-button>
+    <el-dialog 
+    title="Создать аккаунт"
+    :visible.sync="isVisibleRegistrationForm" 
+    center
+    :show-close="false"
```

Comment:
why do you skip close button in dialog in element ui and provide your own instead? i sure it makes extra code and styles